packages/cashier/src/Sass/cashier.scss

```diff
@@ -1122,6 +1122,79 @@
     }
 }
 
+.cashier-recent-transaction-wrapper {
```

Comment:
Why not just create another folder with the `.jsx` file, the `.scss` file and an `index.js` file, see https://github.com/binary-com/deriv-app/pull/3634.

It'll still be accessible across cashier package but this scss file is 1100+ lines long!! 😮 Let's not add to it.

Reply:
I can try, that's a valid point